### PR TITLE
Changes to quickstart suggested by Prem

### PR DESCRIPTION
**Pull request by kmoscoe**

### CODE REVIEW

File: custom_dc/quickstart.md [hunk 1]
```diff
@@ -33,7 +33,7 @@ The "services" Docker container consists of the following Data Commons component
 
 ## Prerequisites
 
-- Obtain a [GCP](https://console.cloud.google.com/welcome) billing account and project.
+- Obtain a [GCP](https://cloud.google.com/docs/get-started) billing account and project.
```
Comment (keyurva) [inline custom_dc/quickstart.md:36]: Can we ensure this link and all external links open in a new browser?

File: custom_dc/quickstart.md [hunk 2]
```diff
@@ -77,7 +77,7 @@ cd website
 
 1. Using your favorite editor, open `custom_dc/env.list`.
 1. Enter the relevant values for `DC_API_KEY` and `MAPS_API_KEY`.
-1. Set the `INPUT_DIR` to the full path to the `sample/` directory. For example if you have cloned the repo directly to your home directory, this would be <code>/home/<var>USERNAME</var>/website/custom_dc/sample/</code>.
+1. Set the `INPUT_DIR` to the full path to the `website/custom_dc/sample/` directory. For example if you have cloned the repo directly to your home directory, this might be <code>/home/<var>USERNAME</var>/website/custom_dc/sample/</code>. (If you're not sure, type `pwd` to get the working directory.)
```
Comment (keyurva) [inline custom_dc/quickstart.md:80]: nit: type `pwd` *in the terminal*

**Review by keyurva — APPROVED** [review]

File: custom_dc/quickstart.md (hunk 1 above)
Comment (kmoscoe) [reply custom_dc/quickstart.md:36]: So Prem only mentioned Console links. Do you think we should make all external doc links open in a new window?

File: custom_dc/quickstart.md (hunk 2 above)
Comment (kmoscoe) [reply custom_dc/quickstart.md:80]: Good point!  Done!

File: custom_dc/quickstart.md (hunk 1 above)
Comment (keyurva) [reply custom_dc/quickstart.md:36]: Yes.

Comment (kmoscoe) [reply custom_dc/quickstart.md:36]: OK, let me do that in another PR, because it will affect the entire doc set.

**Pull request merged**